services/api/src/main.ts

```diff
@@ -32,7 +20,6 @@ export async function bootstrap(): Promise<INestApplication> {
   const logger = app.get<Logger>(Logger);
 
   configureApp(app);
-  await setupMicroservices(app);
```

Comment:
A bit further down there is also `app.startAllMicroservices` which also shouldn't be needed anymore.